editor/scene_tree_dock.cpp

```diff
@@ -3238,10 +3304,22 @@ void SceneTreeDock::_files_dropped(const Vector<String> &p_files, NodePath p_to,
 			menu_properties->reset_size();
 			menu_properties->set_position(get_screen_position() + get_local_mouse_position());
 			menu_properties->popup();
-		} else if (!valid_properties.is_empty()) {
+			return;
+		}
+		if (!valid_properties.is_empty()) {
```

Comment:
Why did you remove `else if`?

Reply:
These two branches both returns now. So the `else` is redundant.